src/sync/build.test.ts

```diff
@@ -26,7 +26,7 @@ describe("sync/builds", () => {
 	const makeExpectedJiraResponse = (builds) => ({
 		builds,
 		properties: {
-			"gitHubInstallationId": DatabaseStateCreator.GITHUB_INSTALLATION_ID
+			gitHubInstallationId: DatabaseStateCreator.GITHUB_INSTALLATION_ID
```

Comment:
Could probably revert this and the following change as these aren't explicitly tied to the PR. Updated these as I initially made changes to these files in my original implementation. That said, they're small enough, and there aren't many changes in this PR, so I think we can leave them.